src/rez/package_test.py

```diff
@@ -209,11 +215,20 @@ def print_header(txt, *nargs):
 
                 print_header("\nRunning test command: %s\n", cmd_str)
 
-            retcode, _, _ = context.execute_shell(
-                command=command,
-                stdout=self.stdout,
-                stderr=self.stderr,
-                block=True)
+            test_result_dir = os.path.abspath('build/rez_test_result' if not test_info["run_in_root"] else '.')
```

Comment:
'build/rez_test_result' should be configurable. As mentioned earlier, we should have a `tests_artifact_path` setting rather than `run_in_root`. Then, to run tests in root for a given package, you do a package-level config override, like so:

```
with scope("config") as c:
    c.tests_artifact_repository = '.'
```

This is more consistent with how settings are managed in other parts of the codebase.

Reply:
agree